style2.css

```diff
@@ -0,0 +1,18 @@
+.oneArea h1 {
```

Comment:
I think this file is not needed. So if you don't mind, pls delete this file.

Reply:
Deleted all not needed file. I’m sorry...